database1/pgQueryTests.js

```diff
@@ -0,0 +1,64 @@
+const pg = require('./pgQueryTests.js')
+const { Client } = require('pg');
+
+const client = new Client({
+  user: 'clairemelbourne',
+  database: 'similar_prod'
+});
+
+client.connect();
+
+//get request from 14000000
+const getOne = (productId, res) => {
+  const priceText = `SELECT price FROM products WHERE productid = ${productId}`;
+  client.query(priceText)
+  .then((result) => {
+    const price = result.rows[0].price
+    const productsText = `SELECT * FROM products WHERE ${getRange(productId, null)} AND price <= ${price + 50} AND price <= ${price - 50} LIMIT 8`;
+    client.query(productsText)
+    .then((result) => {
+      console.log("results of query", result.rows);
+    })
+  })
+  .catch((err) => {
+    console.error(err.stack);
+    client.end();
+  })
+}
+
+const create = (req, res) => {
+  console.log("request body", req.body);
+  const b = req.body
```

Comment:
It's up to you, but I would at least consider using more descriptive variable names. It's hard to know what 'b' or 'text' are meant to represent at a glance. I can tell by reading what comes after them, but if I saw 'b' 300 lines later I'm not sure I'd immediately connect that it's the 'req.body'

Reply:
Fair point-- my reasoning for using b was that the content after it (category, name etc) was what someone would need to know, but on first encountering, it does seem reasonable to keep body so the reader knows where the categories come from without having to trace back! I used text bc I saw it in examples, but I will modify to be more descriptive!